course1/13.popular_package/6.package_buffer/task1.13.6.1/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+)
+
+// getReader takes bytes.Buffer obj and returns bufio.Reader of the given obj
+func getReader(b *bytes.Buffer) *bufio.Reader {
```

Comment:
no use returning pointer to an interface
_An interface in Go holds a pair: the dynamic type and a pointer to the dynamic value. So, returning a pointer to an interface adds an unnecessary level of indirection._